### PR TITLE
add staticcall to cost function

### PR DESCRIPTION
**Pull request by ayrat555**

### CODE REVIEW

File: Paper.tex [hunk 1]
```diff
@@ -1843,7 +1843,7 @@ \subsection{Gas Cost}
 G_{log}+G_{logdata}\times\boldsymbol{\mu}_{\mathbf{s}}[1]+3G_{logtopic} & \text{if} \quad w = \text{\small LOG3} \\
 G_{log}+G_{logdata}\times\boldsymbol{\mu}_{\mathbf{s}}[1]+4G_{logtopic} & \text{if} \quad w = \text{\small LOG4} \\
 C_\text{\tiny CALL}(\boldsymbol{\sigma}, \boldsymbol{\mu}) & \text{if} \quad w = \text{\small CALL} \lor \text{\small CALLCODE} \> \lor \\
-&\quad\text{\small DELEGATECALL} \\
+&\quad\text{\small DELEGATECALL} \lor \text{\small STATICCALL} \\
```
Comment (livnev) [inline Paper.tex]: This looks correct to me.

**Comment by Recmo** [comment]
I was about to open a similar PR. Currently the gas cost for `STATICCALL` is undefined.

**Comment by ayrat555** [comment]
@alexprut can you please take a look?

**Comment by yperbasis** [comment]
Already fixed by PR #796

**Pull request closed without merging**